src/protractor_test/index.bzl

```diff
@@ -0,0 +1,35 @@
+load("@npm_rules_browsers//:protractor/package_json.bzl", protractor = "bin")
+load("@rules_browsers//src/server_test:index.bzl", "server_test")
+
+def protractor_test(name, server, deps, **kwargs):
+    protractor.protractor_test(
+        name = "%s_test" % name,
```

Comment:
Is there a better naming scheme we can have here? This target will end up as `some_name_test_test` all the time, I wonder if we can come up with something more apparent in the target naming